src/App.tsx

```diff
@@ -1,24 +1,195 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  FC,
+  useState,
+  useEffect,
+} from 'react';
 
-const USER_ID = 0;
+import * as todosServices from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Filters } from './types/Filters';
+import { Error } from './components/Error';
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+const USER_ID = 11839;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [response, setResponse] = useState(false);
+  const [isLoading, setIsLoading] = useState<number[]>([]);
```

Comment:
```suggestion
  const [isLoadingTodos, setIsLoadingTodos] = useState<number[]>([]);
```

i think it will be more informative)